graphql_compiler/macros/macro_edge/validation.py

```diff
@@ -61,6 +65,39 @@ def _validate_macro_ast_with_macro_directives(schema, ast, macro_directives):
                 .format(directive_definition.name, len(macro_data)))
 
 
+def _validate_macro_ast_directives(ast, inside_fold_scope=False):
+    """Check that the macro is using non-macro direcives properly."""
+    subselection_inside_fold_scope = inside_fold_scope
+    for directive in ast.directives:
+        name = directive.name.value
+        if name == FilterDirective.name:
```

Comment:
Unify all of these `pass` cases in a single case where you check `if name in <set>`.